dicoogle/src/main/resources/webapp/index-template.html

```diff
@@ -30,14 +30,17 @@
 
     <link rel="stylesheet" href="css/jquery-ui.css">
 
-     <!--Custom JS -->
-     <!--
+    <!-- React Select CSS -->
+    <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
```

Comment:
We prefer keeping these resources locally. Can you add this css file to our css/ folder and import it from there?